### PR TITLE
Small cleanup to README.md

### PR DESCRIPTION
**Pull request by MitchellCash**
- Add the Kodi logo
- Update some markdown formatting

I was unsure where to locate a better quality logo. If someone could point me in the right direction I can update this pull. Thanks :)

### CODE REVIEW

**Comment by mitchellcash** [comment]
@un1versal made some changes in reference to comments made in https://github.com/mitchellcash/xbmc/commit/305965dcd3f68209390553de732be9d834d95499

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-#### About
+![Kodi logo](https://kodi.tv/wp-content/uploads/side-by-side-dark-transparent.png)
```
Comment (un1versal) [inline README.md]: https://github.com/xbmc/xbmc-forum/blob/master/xbmc/images/logo-sbs-black.png

**Comment by mitchellcash** [comment]
@un1versal cleaned up further following your comments :)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-#### About
+![Kodi logo](https://raw.githubusercontent.com/xbmc/xbmc-forum/master/xbmc/images/logo-sbs-black.png)
```
Comment (AlwinEsch) [inline README.md]: Looks good for me.

For the logo you can use this way:
`[Home](http://kodi.tv/)`

with them you have a link to kodi, instead the image.

Comment (un1versal) [inline README.md]: you can do that in 100% markdown thats ugly

**Comment by AlwinEsch** [comment]
@un1versal you are right.
This seems to be better:
`[Home](http://kodi.tv/)`

The last times searched for thinks like this found only the way before :)

**Comment by Razzeee** [comment]
This looks much nicer, thank you!

**Pull request merged**